segtypes/common/code.py

```diff
@@ -36,8 +37,9 @@ def __init__(
             vram_start,
             extract,
             given_subalign,
-            exclusive_ram_id,
```

Comment:
I think bss_size only needs to be a field in code.py (the code group segment type) - and we can override vram_end in just that segment